Description: Fix armhf baseline violation and armel FTBFS caused by NEON usage
Author: Adrian Bunk <bunk@debian.org>
Bug-Debian: https://bugs.debian.org/874220

--- openni2-2.2.0.33+dfsg.orig/ThirdParty/PSCommon/BuildSystem/Platform.Arm
+++ openni2-2.2.0.33+dfsg/ThirdParty/PSCommon/BuildSystem/Platform.Arm
@@ -4,7 +4,7 @@ export GLUT_SUPPORTED=1
 ifeq "$(CFG)" "Release"
 
     # Hardware specifying flags
-    CFLAGS += -Wno-unused-local-typedefs -march=armv7-a -mtune=cortex-a9 -mfpu=neon #-mfloat-abi=softfp #-mcpu=cortex-a8
+    CFLAGS += -Wno-unused-local-typedefs #-march=armv7-a -mtune=cortex-a9 -mfpu=neon #-mfloat-abi=softfp #-mcpu=cortex-a8
 
     # Optimization level, minus currently buggy optimizing methods (which break bit-exact)
     CFLAGS += -O3 -fno-tree-pre -fno-strict-aliasing
@@ -12,6 +12,6 @@ ifeq "$(CFG)" "Release"
     # More optimization flags
     CFLAGS += -ftree-vectorize -ffast-math -funsafe-math-optimizations #-fsingle-precision-constant
 
-    DEFINES += XN_NEON
+    #DEFINES += XN_NEON
     CFLAGS += -flax-vector-conversions
 endif
